integration-tests/test/bridged-tokens.ts

```diff
@@ -55,27 +70,52 @@ describe('Bridged tokens', () => {
       '0x'
     )
     await env.waitForXDomainTransaction(tx, Direction.L1ToL2)
-    expect(await L1__ERC20.balanceOf(l1Wallet.address)).to.deep.equal(
+    expect(await L1__ERC20.balanceOf(env.l1Wallet.address)).to.deep.equal(
       BigNumber.from(999000)
     )
-    expect(await L2__ERC20.balanceOf(l1Wallet.address)).to.deep.equal(
+    expect(await L2__ERC20.balanceOf(env.l2Wallet.address)).to.deep.equal(
       BigNumber.from(1000)
     )
   }).timeout(isLiveNetwork() ? 300_000 : 120_000)
 
-  it('should withdraw tokens from L2', async () => {
+  it('should transfer tokens on L2', async () => {
+    const tx = await L2__ERC20.transfer(otherWalletL1.address, 500)
```

Comment:
This may be more explicity to use `otherWalletL2` even tho both addresses are the same